### DISCUSSION TITLE
I SHIP You 💝  [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#108338] [New to GitHub]
![Header of New to GitHub category check-in shows a purple starry background with shooting stars and clouds next to an octocat in a beret, striped shirt, and red pants holding a stack of books.](https://github.com/community/community/assets/92824518/3ff6f15b-be36-4fac-a2c8-e1bc3ba92918)

Welcome to a special Valentine’s edition of New to GitHub ❤️‍🔥.  Our love for our members in this check-in is extra special and we’ve got GitHub projects to celebrate the extra special someones in your life this holiday. 

### 🌟Community Superstars: A round of applause for these community citizens 👏 
* @yevgenperedkov: Just got started on their developer journey and are in need of a mentor. If you want to spread your GitHub love - [find them in this Discussion](https://github.com/orgs/community/discussions/107543). 
* @smooth-web: Asked a [great question](https://github.com/orgs/community/discussions/106702) about which approach a newbie should take. The more perspectives on this discussion, the merrier - so add yours! 
* @mrpmartin: Is a self-solving superhero 🦸. They did some sleuthing as to why their commits were unverified and came back to share [the answer](https://github.com/orgs/community/discussions/106560#discussioncomment-8390776) with us. Thanks for enriching the community! 

### 💐 Valentine’s Day: Celebrate the holiday GitHub-style ❣️
* **Interactive Valentine’s Day Card**: @dominicarrojado built an app with `Next.js` and `TypeScript` for an interactive card that is sweeter than candy hearts 💌. [Check it out](https://github.com/dominicarrojado/hashtag-interactive-valentines-day-card-app). 
* **Valentine’s Card Maker:** No glitter or card stock required for this card. Build your own Valentine’s card with [this project](https://github.com/sanwebinfo/valentines-day) 😍. 
* **Git-themed Valentine’s Cards**: Cards for a fellow Git lover in your life that are clever and delightful. Thanks @winwardo for creating [this project](https://github.com/Winwardo/git-valentines-cards). 

### ⬆️ Level up: Resources to take your skills to new heights for the new year
* **A complete guide to open source - 100x simpler**: You may have seen @Anmol-Baranwal around the community, he continues his helpfulness by putting together [a guide to contributing to open source](https://dev.to/anmolbaranwal/a-complete-guide-to-open-source-100x-simpler-2d6c). Check it out! 
* **How to code more securely with AI**: From automation to AI, integrating security into the developer workflow has come a long way, enabling us to find vulnerabilities earlier and to bring about a better discourse between security teams and developers. [Watch our video](https://www.youtube.com/watch?v=AF0xIgiQqmY) to hear from security experts on the importance of automation, the impact of AI, and how security teams and developers can work better together.

### 📣 Announcements: The latest updates in the Community 🆕
* **Achievements will no longer be available in the Community**:  In response to this fast growing issue and comments from members around the galaxy brain achievement, we have made the decision to turn off the ability to earn Achievements in our Community. [Read more for details](https://github.com/orgs/community/discussions/106536). 

Thank you everyone who makes this space a sweet one all days of the year 🍫. Here is to falling more in love with GitHub on our learning journeys.

**Comment by Unity-Billal-mesloub** [comment]
Thank you for your comments

**Comment by dominicarrojado** [comment]
Thanks for the special mention of my project! 😊🙏🏻 Happy Valentine's Day ~

**Reply by queenofcorgis** [reply]
Thanks for creating the project 💘  Hope you had a great day!

**Comment by Anmol-Baranwal** [comment]
@queenofcorgis Thank you!
I'm surprised that it reached here. I only posted it on DEV.
Many people have asked me about how to get started in open source and do things properly.
So, I decided to cover everything in a single post. I'm happy that you recommend it. It means a lot :D

**Reply by queenofcorgis** [reply]
@Anmol-Baranwal I'm always lurking around on dev.to and recognized your name so I gave it a read. I thought the content that you wrote would be helpful for a lot of users so I wanted to amplify 🌟  Keep up the great work - I look forward to hopefully seeing more!

**Reply by Anmol-Baranwal** [reply]
I will now be writing on freeCodeCamp and reposting it on Dev.to in most cases. 
Thank you for reading. The Dev community is truly awesome!

**Comment by queenofcorgis** [comment]
Leap into our latest [check-in](https://github.com/orgs/community/discussions/110562) 🐸
